src/Documentation~/compatibility.md

```diff
@@ -0,0 +1,15 @@
+# Compatibility with `Mathf`
+
+Unity Mathematics isn't implemented in the same way as the [`UnityEngine.Mathf`](https://docs.unity3d.com/ScriptReference/Mathf.html) struct in the core Unity engine. Because of this, if your application relies on the specific behaviors of `Mathf`, it might not be possible to use Unity Mathematics in your project. Also, if your project uses `Mathf` functions that have no corresponding Unity Mathematics function, or your code uses external plug-ins that depend on Unity types such as `Vector3`, or `Matrix4x4`, then you shouldn't use Unity Mathematics.
+
+>[!IMPORTANT]
+>For performance reasons, if your project uses the Mono compiler, you should continue to use the mathematical operations in `Mathf`, rather than Unity Mathematics.
+
+## Porting `Mathf` code to Unity Mathematics
+
+If you want to migrate code from `UnityEngine` to `Unity.Mathematics`, you need to make the following changes first:
+
+* `Vector4` to `float4`
+* Multiplications on vector or matrix objects to component multiplication with the `*` operator.
+* Degrees to radians
+* How your code generates random numbers
```

Comment:
I think we should expand here to explain that Unity.Mathematics random number generation is totally user controlled with the `Random` type. By this, I mean that you control the state entirely by having to seed/initialize the `Random` instance and you can draw numbers from the `Random` instance that you initialized. Furthermore, you can have as many `Random` instances as you like.

In fact, the documentation for Unity's Random is kind of like the anti-documentation of Unity.Mathematics Random. https://docs.unity3d.com/ScriptReference/Random.html The two biggest differences would be `static vs instanced` and `Float upper bounds are inclusive`. 

Unity.Mathematics Random is instanced and if you want it to be static, the user has to create it themselves as a static. We don't recommend people doing this unless they know they will only get random numbers from the main thread.

The float upper bounds being inclusive is another big difference. Unity.Mathematics Random is _exclusive_ with the upper bound. Users should be very careful here if they are migrating/converting code that is sensitive to the bounds.